src/object/class_object.c

```diff
@@ -7986,15 +8013,6 @@ classobj_check_index_compatibility (SM_CLASS_CONSTRAINT * constraints, const DB_
 	  ret = SM_CREATE_NEW_INDEX;
 	  return ret;
```

Comment:
Line 7942: the table cell for FK-FK should be updated from shared to error.

Reply:
Ok, goo point, thanks.